pom.xml

```diff
@@ -12,6 +12,7 @@
         <module>readyapi4j-facade</module>
         <module>readyapi4j-groovy-dsl</module>
         <module>readyapi4j-swagger</module>
+        <module>samples</module>
```

Comment:
We talked about removing readyapi4j from folder names. The maven module name (readyapi4j-samples) still contains 'readyapi4j' in it's name.